localstack-core/localstack/services/apigateway/next_gen/execute_api/template_mapping.py

```diff
@@ -50,6 +53,58 @@ class MappingTemplateVariables(TypedDict, total=False):
     stageVariables: dict[str, str]
 
 
+def get_java_formatter(value):
```

Comment:
coming back to the naming, it could be something like `getVTLObject` or something related to "type"? I can't really find any good word to replace "formatter". https://velocity.apache.org/engine/devel/developer-guide.html#how-velocity-works